plugins/ssh/help.md

```diff
@@ -57,17 +65,18 @@ This can then be pasted into the Connection's `key` input field.
 
 ### Actions
 
-#### Run Remote Command
 
-This action is used to run remote command.
+#### Run Remote Command
+  
+Run remote command
```

Comment:
The new tooling is missing the previous toolings functionality to include a prefix and full stop in help md.
```suggestion
This action is used to run remote command.
```
revert this one line.